README.md

```diff
@@ -19,6 +72,7 @@ Coming soon!
 | cert_bucket_name | The name of the AWS S3 bucket where certificates are stored. | string | `cisa-cool-certificates` | no |
 | cool_domain | The domain where the COOL resources reside (e.g. "cool.cyber.dhs.gov"). | string | `cool.cyber.dhs.gov` | no |
 | guac_connection_setup_path | The full path to the dbinit directory where initialization files must be stored in order to work properly (e.g. "/var/guacamole/dbinit"). | string | `/var/guacamole/dbinit` | no |
+| operations_instance_counts | A map specifying how many instances of each type should be created in the operations subnet (e.g. {\ "kali\": 1 }). | map(number) | `{ "kali": 1 }` | no |
```

Comment:
A list of possible keys would be good here.  The list can be appended to as more instance types are supported.

Reply:
Great suggestion, thanks!  Added in https://github.com/cisagov/cool-assessment-terraform/pull/16/commits/3e2cf9971b31be012a77dc0d310486418685af3f.